py/build.gradle

```diff
@@ -10,6 +12,29 @@ ext {
     distDir = file("$buildDir/dist")
 }
 
+def getPythonVersion() {
+    Python p = new Python(project, python.getPythonPath(), python.getPythonBinary())
+    return p.version
+}
+
+def getPythonMajorVersion() {
+    Python p = new Python(project, python.getPythonPath(), python.getPythonBinary())
+    String[] split = p.version.split("\\.")
```

Comment:
Would it be possible to use ```tokenize(".")``` instead of ```split("\\.")```?

Reply:
yup!